ballsdex/packages/trade/cog.py

```diff
@@ -413,6 +427,64 @@ async def history(
         pages = Pages(source=source, interaction=interaction)
         await pages.start()
 
+    @coins.command(name="add")
+    async def coins_add(self, interaction: discord.Interaction, amount: int):
+        """
+        Add coins to your trade proposal
+        """
+        trade, trader = await self.check_trade_errors(interaction)
+        if not trade or not trader:
+            return
+
+        if trade:
+            if amount > await trader.fetch_player_coins():
+                await interaction.response.send_message(
+                    f"You don't have enough {settings.currency_name} to add that amount."
+                )
+            else:
+                await trader.add_coins(amount)
+                await interaction.response.send_message(
+                    f"Added {amount} {settings.currency_name} to your proposal."
+                    f"Total in proposal: {trader.coins} {settings.currency_name}.",
```

Comment:
```suggestion
                    f"Added {amount} {settings.currency_name} to your proposal. "
                    f"Total in proposal: {trader.coins} {settings.currency_name}.",
```
just for correct grammar